services/web-app/components/svg-libraries/action-bar/action-bar.js

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright IBM Corp. 2022, 2022
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import { Code32, Download32 } from '@carbon/icons-react'
+import { IconButton } from '@carbon/react'
+import { pascalCase } from 'change-case'
+import clsx from 'clsx'
+import copy from 'copy-to-clipboard'
+import PropTypes from 'prop-types'
+import React, { useContext, useRef, useState } from 'react'
+
+import { LibraryContext } from '../library-provider'
+import styles from './action-bar.module.scss'
+
+const ActionBar = ({
+  name,
+  friendlyName,
+  source,
+  setIsActionBarVisible,
+  isActionBarVisible,
+  isLastCard,
+  glyphOnly
+}) => {
+  const { site, type } = useContext(LibraryContext)
+  let suffix
+  if (type === 'pictogram') {
+    suffix = ''
+  } else if (glyphOnly) {
+    suffix = 'Glyph'
+  } else {
+    suffix = '32'
+  }
+  const component = `<${pascalCase(friendlyName) + suffix} />`
```

Comment:
rather than relying on this external library to perform conversion, are we able to do use the component's display name? (i.e. `Component.displayName` or `Comonent.name`)

Reply:
I'm not following, where is this information coming from?

Reply:
Synced with Joe, icon components are not including these props so we're stuck with 'change-case' for the time being